src/llmtuner/extras/misc.py

```diff
@@ -5,6 +5,9 @@
 from typing import TYPE_CHECKING, Any, Dict, Optional, Tuple
 from transformers import InfNanRemoveLogitsProcessor, LogitsProcessorList
 
+import logging
+logger = logging.getLogger(__name__)
```

Comment:
Use `from llmtuner.extras.logging import get_logger` and `logger = get_logger(__name__)` instead

Reply:
sure, it is fixed